File: JDBCTest.java
import java.sql.*;
import java.io.*;

public class JDBCTest {
	public static void main(String[] args){
		
		try{
			//先加载jar文件sqljdbc4.jar
			Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver");
			//利用windows账号登陆，不指定账号密码
			String dbURL = "jdbc:sqlserver://0.0.0.0:1433;DatabaseName=test;integratedSecurity=true;";
			Connection con = DriverManager.getConnection(dbURL);
			System.out.println("数据库连接成功");
			
			
			Statement stmt = con.createStatement(); 
			String query = "select * from tb;";
			ResultSet rs = stmt.executeQuery(query); 
			System.out.println("搜索完成");
			
			
			
			while(rs.next()){
				System.out.println(rs.getString("id")+" "+rs.getString("name"));
			}
			con.close();
		}catch(Exception e){
			System.out.println("发生错误");
			StringWriter sw = new StringWriter();   
            PrintWriter pw = new PrintWriter(sw);   
            e.printStackTrace(pw);   

			System.out.println("\r\n");
			System.out.println(sw.toString());
			System.out.println("\r\n");
		}
		
	}
}

File: JDBCTest.scala
import java.sql.DriverManager
import java.sql.Connection
/**
  * Created by Administrator on 2016-12-9.
  */
object JDBCTest {
  def main(args: Array[String]) {
    // connect to the database named "mysql" on the localhost
    val driver = "com.microsoft.sqlserver.jdbc.SQLServerDriver"
    //val driver = "com.microsoft.sqlserver.jdbc.SQLServerDriver"
    val url = "jdbc:sqlserver://0.0.0.0:1433;DatabaseName=test;integratedSecurity=true;"

    // there's probably a better way to do this
    var connection:Connection = null

    try {
      // make the connection
      Class.forName(driver)
      println("class ok")
      connection = DriverManager.getConnection(url)

      // create the statement, and run the select query
      val statement = connection.createStatement()
      val resultSet = statement.executeQuery("SELECT * FROM tb;")
      while ( resultSet.next() ) {
        val id = resultSet.getString("id")
        val name = resultSet.getString("name")
        println(id+" "+name)
      }
      connection.close()
    } catch {
      case e:Any =>  e.printStackTrace
    }

  }
}
